test/common/upstream/subset_lb_test.cc

```diff
@@ -1993,6 +1994,7 @@ TEST_F(SubsetLoadBalancerTest, EnabledScaleLocalityWeights) {
       },
       host_set_, {50, 50});
 
+  common_config_.mutable_locality_weighted_lb_config();
```

Comment:
This change not just fix the test for new implementation. But also make the test closer to the reality.

Because the `localityWeightAware` can only be enabled when the `locality_weighted_lb_config` is set.